src/course/handbook/projects.md

```diff
@@ -1,75 +1,71 @@
-# Project
+# Projects
 
 Each week, you will be split into teams of 4 for weekly projects. These teams will be change throughout the course, giving you a chance to work with every other member of your cohort.
 
-The teams stay together for the following weeks:
+The founders and coders learning environment is all about collaboration, and this is a great opportunity to practice your workflow in a development team.
 
-- First team = weeks 1 & 2
-- Second team = weeks 3 & 4
-- Third team = weeks 5 & 6
-- Fourth team = weeks 7, 8 & 9
-- Fifth team = weeks 10, 11 & 12
+Project teams for each week are posted in your cohort repo, under the pairs-and-teams folder.
 
-Your teams are posted on the wall by the door or can be found in your cohort's repository in your GitHub organisation. Remember to read each other's user manuals when you start working with a new team.
+Remember to read each other’s user manuals when you start working with a new team!
```

Comment:
Did the quote character change here? 👀

Reply:
Weird - it looks more angled on my screen

`'` vs  `’`